### DISCUSSION TITLE
Changelog missing content for @chakra-ui/react@v2.9.1, what's in that release?

### DISCUSSION THREAD

**Discussion by kevinxh** [chakra-ui/chakra-ui#8875] [Q&A] [answered]
I'm here because it appears our system's CI is broken and chasing the errors we have found chakra-ui related errors. We suspect it's related to the latest release that happened in the last 2 days, based on npm version history.

![CleanShot 2024-10-02 at 10 37 12@2x](https://github.com/user-attachments/assets/753a2c2f-a6f7-4ed5-99a1-68274701a357)

It appears there isn't changelog items for the new releases.

![CleanShot 2024-10-02 at 10 35 09@2x](https://github.com/user-attachments/assets/2a73ea38-6f0d-4b90-a8f9-afda7aa3c7b7)

Can someone help me understand what's in the latest release?

**Comment by segunadebayo** [comment, marked as answer]
You can find the changelog here:
https://github.com/chakra-ui/chakra-ui/blob/v2/.changelog/v2.mdx
